src/components/layout/footer.tsx

```
@@ -1,52 +1,81 @@
+              <span className="text-sm font-medium">unreal@outlook.com</span>
+            </div>
           </div>
-          <div className="flex items-center gap-6 text-sm text-white/80">
-            <a href="#" className="transition-colors hover:text-white">
-              Privacy
+
+          {/* Social Media */}
+          <div className="flex items-center gap-4">
+            <a
+              href="#"
+              className="flex h-10 w-10 items-center justify-center rounded-full bg-white text-primary transition-transform hover:scale-110"
```

Biome (2.3.13)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)